tests/pages/base.py

```diff
@@ -5,21 +5,21 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.select import Select
 
-from page import Page, PageRegion
+from pypom import Page, Region
 from regions.newsletter import NewsletterEmbedForm
 
 
 class BasePage(Page):
 
-    _url = '{base_url}/{locale}'
+    URL_TEMPLATE = '/{locale}'
 
-    def __init__(self, base_url, selenium, locale='en-US', **kwargs):
-        super(BasePage, self).__init__(base_url, selenium, locale=locale, **kwargs)
+    def __init__(self, selenium, base_url, locale='en-US', **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
 
     def wait_for_page_to_load(self):
         super(BasePage, self).wait_for_page_to_load()
-        el = self.find_element((By.TAG_NAME, 'html'))
-        self.wait.until(lambda s: 'loaded' in el.get_attribute('class'))
+        self.wait.until(lambda s: self.is_element_present(
```

Comment:
I noticed a stale element exception occurring when loading subsequent pages via a click. This is most likely because the HTML element is found before the new page load starts, but is then stale by the time we check the attribute. This change resolved my stale element exceptions, but I now realise that it could mean the wait condition is met prematurely, though that's also possible with the current wait.